hw/bsp/stlinkv3mini/board.mk

```diff
@@ -1,3 +1,7 @@
+# Only OTG-HS has a connector on this board
+
+REDUCE_SPEED ?= 0
```

Comment:
could you change this option to `SPEED=full` or `SPEED=high` to make it clearer for user.